docs/cmd/iso.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+parent: Command line reference
+nav_order: 4
+---
+
+# coreos-installer iso ignition embed
```

Comment:
This means the page shows up in the index as `coreos-installer iso ignition embed`, but it should probably be overridden to `coreos-installer iso`.  Same for the PXE page.